draft-ietf-quic-applicability.md

```diff
@@ -454,6 +454,21 @@ Some deadlocking scenarios might be resolved by cancelling affected streams with
 STOP_SENDING or RESET_STREAM.  Cancelling some streams results in the connection
 being terminated in some protocols.
 
+# Stream Limit Commitments
+
+QUIC endpoints can manage the cumulative maximum number of streams they would
+allow to be opened using the MAX_STREAMS frame, but there is no mechanism to
+reduce the value. An application that uses QUIC might commit to a number of
+openable streams but require the connection to be closed (for example, a
+scheduled maintenance period). Depending on how an application uses QUIC
+streams, abrupt closure of actively used streams may be undesireable or
+detrimental. In contrast, waiting for an endpoint to exhaust the advertised
+limit may not suit application or operational needs. Applications using QUIC can
+use conservative stream limits and run to completion before enacting an
+immediate close. Alternatively, a graceful close mechanism can be used to
```

Comment:
Do you mean graceful close at the application layer?

Reply:
ah good spot. I lost the context when moving the text to this section, so let me make it clear again.